src/utils/useHasSubgraphSyncErrors.ts

```diff
@@ -13,79 +13,77 @@ type EventData = {
 
 const SLOW_THRESHOLD = 5000
 
-const getBadQueries = (
-  queryCache: QueryCache,
-  eventData: RefObject<EventData>,
-  renderedAt: number,
-) => {
+const getBadQueries = (queryCache: QueryCache, eventData: RefObject<EventData>) => {
   const queries = queryCache.findAll([], {
     // only get subgraph queries that are pending or errored
     predicate: (query) => query.queryKey.at(-1) === 'graph' && query.state.status !== 'success',
   })
-  let slowQueries = 0
-  let errorQueries = 0
+  const now = Date.now()
 
-  queries.forEach((query) => {
-    const { queryHash } = query
-    const isSlow =
-      !!eventData.current?.[queryHash]?.startTime &&
-      (query.state.status === 'loading' || query.state.fetchStatus === 'fetching') &&
-      eventData.current[queryHash].startTime - renderedAt > SLOW_THRESHOLD
+  return queries.reduce(
+    (sum, query) => {
+      // skip if query is not currently active
+      if (query.getObserversCount() === 0) return sum
 
-    const isError = query.state.status === 'error'
-    const isFailedToFetch = query.state.fetchFailureReason instanceof DOMException
+      const { queryHash } = query
+      const isSlow =
+        !!eventData.current?.[queryHash]?.startTime &&
+        query.state.fetchStatus === 'fetching' &&
+        now - eventData.current[queryHash].startTime > SLOW_THRESHOLD
 
-    if (isError) {
-      if (isFailedToFetch) errorQueries += 1
-      else query.invalidate()
-    } else if (isSlow) {
-      slowQueries += 1
-    }
-  })
+      const isError = query.state.status === 'error'
 
-  return { slow: slowQueries, error: errorQueries }
```

Comment:
since it's change to booleans, why not return them as booleans? logic will stay the same in `useHasGlobalError`